build/lib/renderer.js

```diff
@@ -10,9 +10,29 @@ var setMatrix = function (matrix, value) {
     }
 };
 
-var buildCameraProj = function (w, h, camera, proj) {
-    var pw = Math.max(w, h / 3 * 4);
-    var ph = Math.max(h, w / 4 * 3);
+function isMobile() {
+    return /Android|mobile|iPad|iPhone/i.test(navigator.userAgent);
+}
+
+var buildCameraProj = function (input_width, input_height, camera, projMat) {
+    console.log(projMat);
+    var proj = [-2, 0, 0, 0, 0, 2, 0, 0, 0, 0, -1.0000200271606445, -1, 0, 0, -0.02000020071864128, 0];
```

Comment:
this are hardcoded values, but they should be correct for the example setup. The reason i'm providing the proj matrix in this way is because my buildProjectionMatrix is not outputting correctly the array. I will fix this soon.